index.html

```diff
@@ -5,45 +5,364 @@
+   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Lato">
+
   <style>
     .header {
-      text-align: center;
+      text-align: left;
+      margin-top: 70px;
+     
     }
+
+    .income {
+      font-color: #F2998F;
+    }
+
+    .bar {
+          height: 20px;
+          background: #F2998F;
+        }
+      
+    .pager>li.previous>a {
+      color: #000000;
+    }
+
+    .pager>li.next>a {
+      color: #000000;
+    }
+
+   table {
+      color:  #ffffff;
+      margin-top: 50px;
+   
+    }
+
+    th {
+      font-size: 16px;
+    }
+
+    body {
+      background: #357C8D;
+      color: #ffffff;
+      font-family: "Lato";
+
+    }
+
+    h1 {
+      font-family: "Lato";
+      font-size: 30px;
+    }
+    img {margin: 50px 50px;}
+  
+    h2 {
+      font-family: "Lato";
+      font-size: 20px;
+      
+    }
+
+    .table-hover tbody tr:hover td {
+      background-color: #5a5858;
+    
+    }
+
+    a {
+      color: #ffffff;
+
+    }
+
+    .btn.outline {
+      background: none;
+      padding: 6px 6px;
+      margin: 8px 8px;
+
+    }
+
+    .btn-primary.outline {
+      border: 2px solid #ffffff;
+      color: #ffffff;
+    
+    } 
+    
+    .btn-primary:hover {
+      color: #fff;
+    background-color: #5a5858;
+    border-color: #ffffff; 
+    }
+ 
+   
+
   </style>
 </head>
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <div class = "col-sm-7">
+        <h1>A Glance at D.C. Neighborhoods</h1>
+        <br />
+        <p>Washington D.C.has changed rapidly in the past decade, with more young professionals moving in certain parts of the District. Using data from <a href="http://www.neighborhoodinfodc.org" target="_blank"><u><b>Neighborhood Info DC</b></u></a>, we can examine exactly how average family income and poverty rates vary among neighborhoods within the District.</p>
+        <div class="controls section">
+          <div class="btn-group" role="group" aria-label="Sort">
+            <h2>filter:</h2> 
+            <button type="button" class="btn btn-primary outline" id="filter-poverty">Poverty Rate>20%</button> 
+            <button type="button" class="btn btn-primary outline" id="filter-income">6-figure income</button>  
+          </div>
+          <br />
+
+          <div class="btn-group" role="group" aria-label="Filter">
+            <h2>sort:</h2> 
+            <button type="button" class="btn btn-primary outline" id="sort-desc-poverty">Poverty Rate</button>
+            <button type="button" class="btn btn-primary outline" id="sort-desc-income">Average Family Income</button>    
+          </div>
+        </div>
+      </div>
+      <div class = "col-sm-5">
+        <img src = "map_washington-dc.gif" alt="D.C. neighborhood map" style="width: 300px; height: 330px;">
```

Comment:
This is a bit of a weird one, but we should rewrite this like:

`<img src = "map_washington-dc.gif" alt="D.C. neighborhood map" width="300" height="330">`

We can set the `width` and `height` attributes of `img` elements directly in order to tell the browser the preferred size. We can then also add size styles in the CSS (but preferably not in a `style` attribute) to do things like set `max-width: 100%;`, which says in essence, "This image can be as big as it likes, as long as it's not bigger than its container."